scripts/us_ca_chhs/healthcare_facility_bed_counts/new_schema.mcf

```diff
@@ -0,0 +1,170 @@
+# -----------------New Schema------------------------
```

Comment:
rm this line

Reply:
And put the rest of this file in the corresponding file in the top level schema/ directory